core/src/main/java/de/danoeh/antennapod/core/feed/FeedItem.java

```diff
@@ -388,6 +391,22 @@ public boolean hasChapters() {
         return hasChapters;
     }
 
+    public void setAutoDownload(boolean autoDownload) {
+        this.autoDownload = autoDownload;
+    }
+
+    public boolean getAutoDownload() {
+        return this.autoDownload;
+    }
+
+    public boolean isAutoDownloadable() {
+        return this.hasMedia() &&
+                false == this.getMedia().isPlaying() &&
```

Comment:
so, just to be pedantic, why "false == this.getMedia().isPlayer()" instead of "!this.getMedia().isPlayer()"?  I've never seen it done this way before.